src/components/histogram/histogram.tsx

```diff
@@ -0,0 +1,18 @@
+import {Progress} from './progress'
+
+type Props = {
+  data: number[]
+}
+
+export const Histogram = ({ data }: Props) => {
+  const min = 10
+  const max = Math.ceil(Math.max.apply(null, data)*1.2)
+  return(
+    <div className="columns-1 justify-left m-2 bg-blue-300 text-sm">
```

Comment:
Style edit - remove background color here:
```suggestion
    <div className="columns-1 justify-left m-2 text-sm">
```